server/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -208,7 +232,7 @@ int getSuccessfulShards() {
             new ObjectParser<>(SnapshotInfoBuilder.class.getName(), true, SnapshotInfoBuilder::new);
 
     private static final ObjectParser<ShardStatsBuilder, Void> SHARD_STATS_PARSER =
-        new ObjectParser<>(ShardStatsBuilder.class.getName(), true, ShardStatsBuilder::new);
+            new ObjectParser<>(ShardStatsBuilder.class.getName(), true, ShardStatsBuilder::new);
```

Comment:
NIT: noisy indent

Reply:
roll on autoformatting. Reverted in f025b31f08.